zhaquirks/tuya/builder/__init__.py

```diff
@@ -30,8 +30,16 @@
 from zhaquirks.tuya.mcu import DPToAttributeMapping, TuyaMCUCluster, TuyaOnOffNM
 
 
+class TuyaIASContact(IasZone, TuyaLocalCluster):
```

Comment:
nit: For zigpy, we use `IasZone`. Here, we use `IAS` in upper-case. Do we want to change that to `Ias` to align with zigpy or do you prefer to leave it as-is?